recipe/patches/0004-Remove-all-dependencies-from-setup.py.patch

```diff
@@ -1,30 +1,34 @@
-From bf491f11f47e136e297d654ea1c9b9229d76e706 Mon Sep 17 00:00:00 2001
+From 819d30cab089b9c352169942008753c409e6e93d Mon Sep 17 00:00:00 2001
 From: Kai Fricke <kai@anyscale.com>
 Date: Fri, 3 Dec 2021 10:55:23 -0800
 Subject: [PATCH 4/6] Remove all dependencies from setup.py
 
 Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
 ---
- python/setup.py | 52 ++++++++-----------------------------------------
- 1 file changed, 8 insertions(+), 44 deletions(-)
+ python/setup.py | 58 ++++++++-----------------------------------------
+ 1 file changed, 9 insertions(+), 49 deletions(-)
 
 diff --git a/python/setup.py b/python/setup.py
-index 7ebc0a894..f70018f27 100644
+index b24a9537e..3aa990703 100644
 --- a/python/setup.py
 +++ b/python/setup.py
-@@ -184,25 +184,11 @@ ray_files += [
+@@ -185,29 +185,12 @@ ray_files += [
  # in this directory
  if setup_spec.type == SetupType.RAY:
      setup_spec.extras = {
+-        "data": [
+-            "pandas",
+-            "pyarrow>=4.0.1",
+-            "fsspec",
+-        ],
 -        "default": [
--            "aiohttp",
+-            "aiohttp >= 3.7",
```

Comment:
this change should be reflected in `meta.yaml` dependencies of `ray-core`

Reply:
Fixed.